src/NHibernate.Test/NHSpecificTest/NH1192/Fixture.cs

```diff
@@ -8,8 +8,7 @@ public class Fixture : BugTestCase
 	{
 		protected override bool AppliesTo(Dialect.Dialect dialect)
 		{
-			return !(dialect is Oracle8iDialect) &&
-			       !(dialect is AbstractHanaDialect);
+			return !(dialect is Oracle8iDialect);
```

Comment:
Does not fail on my side too. Am I missing something?

Reply:
I probably excluded the test at first before I implemented the boolean operators and forgot to include it again afterwards.